### ISSUE TITLE
Inject nodes trigger at UTC time instead of local time

### ISSUE THREAD

**Issue by renne** [ct-Open-Source/ct-Smart-Home#15]
Hi,

inject nodes trigger at UTC time instead of local time.
System is Debian 9 Stretch in a KVM virtual machine. Debian timezone and `date` output are correct:
```
renne@Homematic:~$ cat /etc/timezone
Europe/Berlin
renne@Homematic:~$ date
Mo 1. Jul 20:55:49 CEST 2019
```
What' going wrong here?

**Comment by renne** [comment]
Adding `process.env.TZ = "Europe/Berlin";` to `settings.js` solved the problem. I suggest to copy the value from `/etc/timezone` on the host to `settings.js` in the docker container when running the setup script.

**Comment by KarstenDE** [comment]
I found that a docker mapping should help:
`docker run -v /etc/localtime:/etc/localtime`

Can this be integrated easily?

**Comment by merlinschumacher** [comment]
The problem is, that the /etc/localtime solution isn't reliable. The file isn't always present. But i will try to find a solution.

**Comment by merlinschumacher** [comment]
Fixed in 1.2
